src/components/headings/H5.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../styles/variables';
+
+h5.bdl-H5 {
+    @include font-family;
+
+    color: $bdl-gray-62;
+    font-size: 11px;
+    font-weight: normal;
+    letter-spacing: .6px;
+    line-height: 16px;
+    text-transform: uppercase;
```

Comment:
Get rid of this uppercase, headings should not dictate uppercasing, that will be in the localized text if needed.

Reply:
Yeah, debating on that too and remember about the locale thing but it's also part of the BDL, so I am keeping it here.